Jenkinsfile-smoke.groovy

```diff
@@ -2,7 +2,7 @@
  * Requires: https://github.com/RedHatInsights/insights-pipeline-lib
  */
 
-@Library("github.com/RedHatInsights/insights-pipeline-lib") _
+@Library("github.com/RedHatInsights/insights-pipeline-lib@customConfig") _
```

Comment:
You don't need to use @customConfig branch anymore. The PR was merged

Reply:
OK, I will remove this then